src/common/utils/delete_documents.py

```diff
@@ -5,32 +5,39 @@
 from storage.data_source_class import DataSource
 
 
-def delete_list_recursive(value: Union[list, dict], data_source: DataSource):
+def _delete_list_recursive(value: Union[list, dict], data_source: DataSource):
```

Comment:
are the functions that starts with "_" not meant to be used outside this file?

Reply:
Yes. I think that is a python naming convention. Though now that I googled it, it seems to be used mainly for classes and it should actually be 2 underscores, not just one. Still, I've seen it used in the same way as I have now several places in dmss, so I think it is fine.